modules/microshift-making-storage-migration-request.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/microshift-storage-migration.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-making-storage-migration-request_{context}"]
+= Making a storage migration request
+
+Storage migration is the process of updating stored data to the latest storage version, for example from `v1beta1` to `v1beta2`. To update your storage version, use the following procedure.
```

Comment:
Maybe add a line here that says:

The status of the `StorageVersionMigration` resource will state a success or failure of the request. Most times a failure is due to a misnamed group or resource, otherwise if a migration fails it'll most likely be due to an incompatibility between versions.